src/consentManager.ts

```diff
@@ -168,3 +168,18 @@ export const BackendSyncOption = makeEnum({
 /** Override type */
 export type BackendSyncOption =
   typeof BackendSyncOption[keyof typeof BackendSyncOption];
+
+/**
+ * Whether or not to run cross-domain sync
+ */
+export const LocalSyncOption = makeEnum({
+  /** comparable to 'on' -- use private consent sync only */
+  Private: 'private',
```

Comment:
Private is the default and equivalent to on. We should encode this as 'on' so that customers don't get on->private mutations show up in their diff which may be confusing